src/App.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+import { isCompositeComponentWithType } from 'react-dom/test-utils'
+import { InputTask } from './components/InputTask'
+import { RadioForm } from './components/RadioForm'
+import { TaskList } from './components/TaskList'
+import { Title } from './components/Title'
+import { TaskType } from './types/task/task';
+import { OnChangeTaskTextType } from './types/task/on-change-task-text';
+import { AddTaskType } from './types/task/add-task';
+import { CheckTaskStatusType } from './types/task/chek-task-status';
+
+export const App = () => {
+  const [tasks, setTasks] = useState<TaskType[]>([]);
+  const [taskText, setTaskText] = useState<string>('');
+  const [finish, setFinish] = useState<boolean>(false);
+  const onChangeTaskText: OnChangeTaskTextType = (e: any) => setTaskText(e.target.value);
+  const addTask: AddTaskType = () => {
+    if (taskText === '') return;
+    const task = {
+      text: taskText,
+      status: finish
+    }
+    const newTask = [...tasks, task];
+    setTasks(newTask);
+    setTaskText('');
+  }
+  const checkTaskStatus: CheckTaskStatusType = (value: boolean) => {
```

Comment:
明示的に定数に型をつけても良いですが、あまり見ない印象です。
ここでしたら、シンプルに以下のようにする方が良く見る感じです。
```suggestion
  const checkTaskStatus = (value: boolean): string => {
```
作業中 or 完了という文字列を返すような型定義をすると、もっと良いです。

Reply:
以下のコミットにて修正をいたしました！
f59901c